core/tee/tee_cryp_utl.c

```diff
@@ -17,41 +17,10 @@
 
 TEE_Result tee_hash_get_digest_size(uint32_t algo, size_t *size)
 {
-	switch (algo) {
-	case TEE_ALG_MD5:
-	case TEE_ALG_HMAC_MD5:
-		*size = TEE_MD5_HASH_SIZE;
-		break;
-	case TEE_ALG_SHA1:
-	case TEE_ALG_HMAC_SHA1:
-	case TEE_ALG_DSA_SHA1:
-		*size = TEE_SHA1_HASH_SIZE;
-		break;
-	case TEE_ALG_SHA224:
-	case TEE_ALG_HMAC_SHA224:
-	case TEE_ALG_DSA_SHA224:
-		*size = TEE_SHA224_HASH_SIZE;
-		break;
-	case TEE_ALG_SHA256:
-	case TEE_ALG_HMAC_SHA256:
-	case TEE_ALG_DSA_SHA256:
-		*size = TEE_SHA256_HASH_SIZE;
-		break;
-	case TEE_ALG_SHA384:
-	case TEE_ALG_HMAC_SHA384:
-		*size = TEE_SHA384_HASH_SIZE;
-		break;
-	case TEE_ALG_SHA512:
-	case TEE_ALG_HMAC_SHA512:
-		*size = TEE_SHA512_HASH_SIZE;
-		break;
-	case TEE_ALG_SM3:
-	case TEE_ALG_HMAC_SM3:
-		*size = TEE_SM3_HASH_SIZE;
-		break;
-	default:
+	size_t digest_size = TEE_HASH_GET_DIGEST_SIZE(algo);
+	if(digest_size == 0)
```

Comment:
@aschwarzkopf-phytec, minor comments.
Empty space between local variable definitions and instructions.
Prefer `if (!value)` over `if(value == 0)`, and same `if (value)` over `if (value != 0)`, unless a very good reason.
Also add an empty line before `*size = digest_size;`.

Reply:
Ok